module2.js

```diff
@@ -124,7 +150,12 @@ function question7() {
 */
 
 function question8() {
-  // TODO
+  fetch('https://example.com', {
+    method: 'POST',
+    headers: {
+      "Content-Type": "application/json"
+    }
+  })
```

Comment:
The test didn't require the content type to be set, and it's a bit odd to say you're sending JSON but not actually provide a `body`! This works, though, so you get the point!